src/components/App/AppHead.tsx

```diff
@@ -256,6 +256,9 @@ export const AppHead = (props: IProps) => {
             <LinearProgress variant="determinate" value={complete} />
           </div>
         )}
+        {(!busy && !doSave) || complete !== 0 || (
```

Comment:
There is a determinate LinearProgress just above that will show if complete <> 0 or 100. This will show both?

Reply:
No, what happens is it shows the determinant as long as it has a value, the indeterminant doesn't show. When the determinant gets reset but the program is still busy, the in determinant shows.